README.md

```diff
@@ -518,6 +518,7 @@ Table of Contents
   * [auth0.com](https://auth0.com/) — Hosted free for development SSO. Up to 2 social identity providers for closed-source projects.
   * [Authress](https://authress.io/) — Authentication login and access control, unlimited identity providers for any project. Facebook, Google, Twitter and more. First 1000 API calls are free.
   * [Authy](https://authy.com) - Two-factor authentication (2FA) on multiple devices, with backups. Drop-in replacement for Google Authenticator. Free for up to 100 successful authentications.
+  * [BLST Security](https://blstsecurity.com) — BLST CLI tool takes your HTTP logs, uses them to map your API flows and find risks — [FireCracker](https://blstsecurity.com/firecracker)
```

Comment:
It's not clear to me if this CLI interacts with the SaaS or if its purely local? If purely local it's not suitable for this list.